dll/shellext/mydocs/mydocs.rc

```diff
@@ -16,6 +16,9 @@
 
 LANGUAGE LANG_NEUTRAL, SUBLANG_NEUTRAL
 
+IDI_MYDOCS ICON "res/mydocs.ico"
+IDI_MYPICS ICON "res/mypics.ico"
```

Comment:
I wonder if it's possible to include these icons directly from shell32 module without duplicating them in the source tree. 🤔

Reply:
As I see, it's linked to shell32 directly:
https://github.com/reactos/reactos/blob/bbbcd8ed685dd68207eb2e17b18e297fccc54598/dll/shellext/mydocs/CMakeLists.txt#L22
so it might be possible (if it is at all), but I also wonder how to do that, since I did never see anything like this.
As I opened MS mydocs.dll from 2k3 in Resource Hacker 4.7.34 from Rapps, those icon resources are not referred to shell32 at all. Otherwise, I'd see the info about this I think. See the last screenshot in description.

Reply:
Maybe I can include shell32.rc and then use their IDs, but from other side, we don't need an inclusion of all other 235 icons from shell32, so...

Reply:
I'm only asking about using two icons from there without duplicating them **in the source tree** (duplication in the final binary doesn't count, as it's expected)